25-evaluation-3/PRESENTATION.md

```diff
@@ -121,8 +121,8 @@ solve a specific problem.
 
 ## Notes
 
-You have access to your own notes in paper format. Your notes must fit on **four
-sheets of paper (recto)**.
+You have access to your own notes in paper format. Your notes must fit on **4
+(1 + 1.5 + 1.5) sheets of paper (recto)**.
```

Comment:
I am not sure I understand this

Reply:
The same as my previous comment: 1 sheet from the first evaluation, 1.5 from the second and 1.5 new sheets of paper for the last evaluation, a total of 4 sheets for the exam.